addon/fragment.js

```diff
@@ -126,7 +126,12 @@ var Fragment = Model.extend(Ember.Comparable, Ember.Copyable, {
   },
 
   toStringExtension: function() {
-    return 'owner(' + get(internalModelFor(this)._owner, 'id') + ')';
+    let owner = internalModelFor(this)._owner;
+    if (owner) {
+      return 'owner(' + get(owner, 'id') + ')';
+    } else {
+      return '';
+    }
```

Comment:
I actually found the problem with the combination of Ember-Data 2.4 and Ember 2.5. It was triggering a deprecation from Ember-Data using Ember.merge which was causing errors in this function. The problem doesn't seem to exist with Ember-Data 2.5 but I think this is good to prevent crashes.